src/libraries/System.Runtime/tests/System.IO.Tests/StreamWriter/StreamWriter.StringCtorTests.cs

```diff
@@ -31,6 +28,7 @@ public static void EmptyPath_ThrowsArgumentException()
             AssertExtensions.Throws<ArgumentException>("path", () => new StreamWriter(""));
             AssertExtensions.Throws<ArgumentException>("path", () => new StreamWriter("", new FileStreamOptions()));
             AssertExtensions.Throws<ArgumentException>("path", () => new StreamWriter("", true));
+            AssertExtensions.Throws<ArgumentException>("path", () => new StreamWriter("", true, null));
             AssertExtensions.Throws<ArgumentException>("path", () => new StreamWriter("", true, Encoding.UTF8));
```

Comment:
Line 31 is not really adding value because this test is meant to use valid arguments for parameters other than `path`. Can you please add a new test dedicated to validating that null Encoding is allowed and do the same for the Stream overlaods, in this class [StreamWriter.CtorTests.cs](https://github.com/dotnet/runtime/blob/46c9a4fff83f35ec659e6659050440aadccf3201/src/libraries/System.Runtime/tests/System.IO.Tests/StreamWriter/StreamWriter.CtorTests.cs).